web/skins/classic/views/js/montage.js

```diff
@@ -1086,6 +1086,25 @@ function monitorsSetScale(id=null) {
   }
 }
 
+function changeMonitorRate() {
+  const rate = $j('#changeRate').val();
+  monitorsSetRate(rate);
+  setCookie('zmMontageRate', rate);
+}
+
+function monitorsSetRate(fps, id=null) {
+  if (id) {
+    var currentMonitor = monitors.find((o) => {
+      return parseInt(o["id"]) === id;
+    });
+    currentMonitor.streamCommand({command: CMD_MAXFPS, maxfps: fps});
```

Comment:
It would be more correct to write:
`currentMonitor.setMaxFPS(fps);` but due to #4045 this code will not work.